### ISSUE TITLE
trend fails with PyQt4

### ISSUE THREAD

**Issue by cpascual** [taurus-org/taurus_pyqtgraph#26]
To reproduce:
```console
$ python -m taurus.qt.qtgui.tpg.trend 
MainThread     INFO     2019-07-24 18:34:25,837 TaurusRootLogger: Using PyQt4 (v4.11.4 with Qt 4.8.7 and Python 2.7.13)
MainThread     INFO     2019-07-24 18:34:25,955 taurus.qt.qtgui.icon.icon: Setting Tango icon theme (from /nfs/home/cpascual/src/taurus/lib/taurus/qt/qtgui/icon/)
MainThread     INFO     2019-07-24 18:34:26,184 TaurusRootLogger: Plugin "taurus_pyqtgraph" loaded as "taurus.qt.qtgui.tpg"
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/nfs/home/cpascual/src/taurus_pyqtgraph/taurus_pyqtgraph/trend.py", line 280, in <module>
    trend_main(models=('eval:rand()', 'sys/tg_test/1/ampli'))
  File "/nfs/home/cpascual/src/taurus_pyqtgraph/taurus_pyqtgraph/trend.py", line 254, in trend_main
    w = TaurusTrend()
  File "/nfs/home/cpascual/src/taurus_pyqtgraph/taurus_pyqtgraph/trend.py", line 96, in __init__
    self.restoreState, 'state')
  File "/nfs/home/cpascual/src/taurus/lib/taurus/qt/qtcore/configuration/configuration.py", line 307, in registerConfigProperty
    return self._registerConfigurableItem(p, name=name)
  File "/nfs/home/cpascual/src/taurus/lib/taurus/qt/qtcore/configuration/configuration.py", line 335, in _registerConfigurableItem
    if name in self.__configurableItemNames:
  File "/nfs/home/cpascual/src/taurus_pyqtgraph/taurus_pyqtgraph/trend.py", line 145, in __getattr__
    raise AttributeError()
AttributeError
```

**Comment by cpascual** [comment]
It fails with both py2 and py3  when using PyQt4, but it works when using PyQt5

**Comment by cpascual** [comment]
Fixed in v0.2.3a (74c7c9d73883c6c68d38e2b66e0130fad9c717c0 )
